docs/blog/2022-07-20-vex-silence-false-positives.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "VEX: Permanently Silence False Positives with Vulnerability EXchange"
+description: "VEX (Vulnerability EXchange) is a new standard developed by the NTIA to exchange information about which vulnerabilities are actually exploitable in a product."
+slug: vex-silence-false-positives
+date: 2022-05-25T01:37
+keywords: [vex, sbom, "false positive", vulnerability]
+tags: [malware, dependencies, vulnerability, sbom, vex]
+authors: [alex]
+---
+
+<!--
+  ~ Copyright by LunaSec (owned by Refinery Labs, Inc)
+  ~
+  ~ Licensed under the Creative Commons Attribution-ShareAlike 4.0 International
+  ~ (the "License"); you may not use this file except in compliance with the
+  ~ License. You may obtain a copy of the License at
+  ~
+  ~ https://creativecommons.org/licenses/by-sa/4.0/legalcode
+  ~
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+-->
+
+## VEX: Permanently Silence False Positives with Vulnerability EXchange
+
+As we highlighted in our post on False Positives in Vulnerability Scanning, the output of vulnerability assessment tools like npm audit can be cumbersome because they lack context. VEX (Vulnerability EXchange) is a new standard developed by the NTIA to exchange information about which vulnerabilities are actually exploitable in a product.
+
+<!--truncate-->
+
+### The Problem: False Positives
+
+npm audit is a good tool for checking your javascript dependencies against known vulnerabilities. However, it commonly results in alerts for vulnerabilities that are inaccessible at runtime or otherwise irrelevant.[^1] This combined with the fact that npm audit is run after install by default can result in users ignoring the output because the Signal to Noise Ratio is not high enough.[^2]
```

Comment:
caps. Npm audit *and other tools like it*.  Dependabot has these same problems. We can chat about the whole ecosystem, not just audit